QEfficient/generation/text_generation_inference.py

```diff
@@ -98,7 +98,7 @@ def latency_stats_bertstyle(
 
 
 def get_compilation_batch_size(qpc_path: str):
-    qpc_base_path = os.path.dirname(qpc_path)
+    qpc_base_path = os.path.dirname(qpc_path.rstrip("/"))
```

Comment:
Now, does it work for both with "/" and without?

Reply:
yes